set-search-api-client/src/main/java/eu/europeana/set/search/connection/HttpConnection.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * HttpConnector.java - europeana4j
+ * (C) 2011 Digibis S.L.
+ */
+package eu.europeana.set.search.connection;
+
+import java.io.IOException;
+
+import org.apache.commons.httpclient.DefaultHttpMethodRetryHandler;
+import org.apache.commons.httpclient.HttpClient;
+import org.apache.commons.httpclient.methods.GetMethod;
+import org.apache.commons.httpclient.params.HttpMethodParams;
+
+
+/**
+ * The class encapsulating simple HTTP access.
+ *
+ * @author Sergiu Gordea, Roman Graf
+ */
+public class HttpConnection {
```

Comment:
please update this class with the required changes for http client version update